src/tests/roc_audio/test_resampler.cpp

```diff
@@ -149,6 +149,7 @@ TEST(resampler, upscaling_twice_single) {
     }
 }
 
+#ifndef ROC_TARGET_DARWIN
 // Check upsampling quality and the cut-off band with white noise.
 TEST(resampler, upscaling_twice_awgn) {
```

Comment:
I think we want to ignore the test using IGNORE_TEST instead of completely hiding in.

Reply:
Fixed